### PR TITLE
Rename, refactor the codebase

### LINKED ISSUE #84: Renaming components in yajsapi
ensure names of classes and methods are consistent with what we have in yapapi

### PR DESCRIPTION
**Pull request by mdtanrikulu**
Resolves #84.

### CODE REVIEW

File: yajsapi/executor/index.ts [hunk 1]
```diff
@@ -170,12 +73,22 @@ export class _BufferItem {
 type D = "D"; // Type var for task data
 type R = "R"; // Type var for task result
 
-export class Engine {
+export type ExectorOpts = {
```
Comment (azawlocki) [inline yajsapi/executor/index.ts:76]: why not `ExecutorOpts`?

**Review by azawlocki — APPROVED** [review]
I've read it all and it looks good as far as I can tell. Haven't run it though.

File: yajsapi/executor/index.ts (hunk 1 above)
Comment (mdtanrikulu) [reply yajsapi/executor/index.ts:76]: oh typo :)  thanks for catching it.

**Review by mfranciszkiewicz — APPROVED** [review]

File: yajsapi/executor/strategy.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  }
+
+  async score_offer(offer: OfferProposal): Promise<Number> {
+    const linear: ComLinear = new ComLinear().from_properties(offer.props());
+
+    if (linear.scheme.value != BillingScheme.PAYU) {
+      return SCORE_REJECTED;
+    }
+
+    if (linear.fixed_price > this.max_fixed) return SCORE_REJECTED;
```
Comment (filipgolem) [inline yajsapi/executor/strategy.ts:54]: This is not related to renaming, but it's risky to reject offers with an arbitrarily chosen `0.05` initial price. Is this value high or low?

**Review by filipgolem — APPROVED** [review]

File: yajsapi/executor/strategy.ts (hunk 2 above)
Comment (mdtanrikulu) [reply yajsapi/executor/strategy.ts:54]: This seems important, I suggest create an issue for it.

**Pull request merged**